modules/docs/arrow-docs/docs/docs/effects/fiber/README.MD

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
You said you're not publishing the docs yet but this is docs. This is gonna be automatically deployed as soon as Ank runs. Is it intended?

Reply:
With `not publishing` I meant not add a menu item to navigate to `Fiber`. I should've been more specific. So the doc is published but it can not be found by navigating documentation.